### DISCUSSION TITLE
querylog_interval not work

### DISCUSSION THREAD

**Discussion by lyc0221** [AdguardTeam/AdGuardHome#3774] [Q&A]
I seted `querylog_interval: 2160h` , but it does not work. It still poll everyday, and it only saves 2 days. how to do ?

**Comment by ainar-g** [comment]
Which version are you using?  Did you stop AGH before editing the configuration file?

**Reply by lyc0221** [reply]
version: `v0.107.0-b.13`

I updated from UI and restart the service after updated, but it doesn't still work.

**Reply by ainar-g** [reply]
@lyc0221, we were able to reproduce the issue, thanks!  See #3781 for progress.
